app/build.gradle.kts

```diff
@@ -30,12 +32,13 @@ android {
             // Optional, specify signing versions used
             enableV1Signing = true
             enableV2Signing = true
-        }
+        }*/
     }
 
     buildTypes {
         release {
-            signingConfig =signingConfigs.getByName("release")
+            // Temporarily removed signingConfig for debug builds
+            // signingConfig = signingConfigs.getByName("release")
```

Comment:
⚠️ This will result in failing the github workflow for release build. You can simply create a keystore file and make your own release build!